build-system/server/new-server/transforms/transform.ts

```diff
@@ -23,12 +23,14 @@ import transformCss from './css/css-transform';
 
 const argv = minimist(process.argv.slice(2));
 const FOR_TESTING = argv._.includes('integration');
+const HOST = argv.host ? argv.host : '0.0.0.0';
```

Comment:
nit: Could this be simplified as follows?
```suggestion
const HOST = argv.host ?? '0.0.0.0';
```